styles/style.css

```diff
@@ -0,0 +1,216 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+:root {
+  --head1: 55px;
+  --head3: 24px;
+  --heroDesc: 16px;
+  --head4: 13px;
+  --link: 16px;
+  --head2: 32px;
+  --jobTitle: 14px;
+  --black: #272a31;
+  --orange: #ec5242;
+  --gray: #d3d3d3;
+  --white: #fff;
+  --dark: #000;
+  --lightGray: #373a41;
+}
+
+body {
+  font-family: 'Lato', sans-serif;
+  font-family: 'Poppins', sans-serif;
+}
+
+.mobile-menu,
+.mobile {
+  display: none;
+}
+
+a {
+  text-decoration: none;
+}
+
+.desktop nav li a {
+  color: var(--black);
+}
+
+header {
+  display: flex;
+  background: var(--white);
+  min-height: 15vh;
+  flex-direction: column;
+  box-shadow: 0 0 12px 0 rgba(0, 0, 0, 0.12);
+  position: relative;
+}
+
+nav {
+  width: 70%;
+  margin: auto;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+nav ul {
+  display: flex;
+  gap: 32px;
+  color: var(--lightGray);
+  font-size: var(--link);
+}
+
+header div {
+  display: flex;
+  justify-content: center;
+  min-height: 5.3vh;
+  background: var(--black);
+  align-items: center;
+}
+
+header div ul {
+  display: flex;
+  gap: 16px;
+  flex: 1;
+  align-items: center;
+}
+
+footer div {
+  display: flex;
+  width: 60%;
+  margin: auto;
+  gap: 16px;
+  justify-content: center;
+  align-items: center;
+}
```

Comment:
@Shedrack-Sunday 
![Screenshot 2023-04-18 170517](https://user-images.githubusercontent.com/115831124/232772053-82eb25b7-f5a9-40d0-8952-8c476b84be0e.png)
as you can see micro verse said you can personalize it on your own so, in my opinion, no need for this to change its IP on your own and other demands you required I have done all buttons and all other requirements. Otherwise, if new reviewer I know the need to always require changes and make things tough so all other requirements are done. But thankyou for such consideration.